internal/logsink/logsink.go

```diff
@@ -244,6 +248,13 @@ func textPrintf(m *Meta, textSinks []Text, format string, args ...any) (n int, e
 	}
 	buf.WriteString("] ")
 
+	if m.Prefix != nil {
+		buf.WriteString(*m.Prefix)
+		if !strings.HasSuffix(*m.Prefix, " ") {
+			buf.WriteString(" ")
+		}
```

Comment:
I think it makes more sense to do this once in SetLogPrefix, as the original did.

Reply:
The space separator is not necessarily desired for structured sinks, which may choose to put the prefix in its own field.